GameServer/Logic/Maps/SummonersRift.cs

```diff
@@ -186,12 +229,12 @@ public override Target GetRespawnLocation(int team)
             switch (team)
```

Comment:
I think this whole part should be in a dictionary, so something like
```CS
private const Dictionary<int, Target> _spawnsByTeam = new Dictionary<int, Target>
{
    {0, new Target(25.90f, 280)},
    {1, new Target(14119, 14063)}
}
```
and then you can just do
```CS
return _spawnsByTeam[team]
```